.github/workflows/Pipeline.yml

```diff
@@ -52,7 +53,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - run: echo "OSNAME=$(echo ${{ matrix.container }} | sed -e 's|:|-|' -e 's|/stage3||')" >> ${GITHUB_ENV}
+      - run: echo "OSNAME=$(echo ${{ matrix.container }} | sed -e 's|:|-|' -e 's|/.*||')" >> ${GITHUB_ENV}
```

Comment:
👍

Reply:
Actually the two tests that failed are known to be flaky. I'm going to merge this so we can unblock your changes that depend on this.